compiler/src/dmd/cxxfrontend.d

```diff
@@ -475,6 +475,12 @@ bool equivalent(Type src, Type t)
     return dmd.typesem.equivalent(src, t);
 }
 
+Type arrayOf(Type type)
```

Comment:
Would be good to add a link test to cxxfrontend.cc.  I think you are also missing a header entry to `mtype.h`.

Reply:
Done.